lib/resolver/homebrew-file.js

```diff
@@ -3,6 +3,6 @@ import relativeFile from './relative-file.js';
 export default function ({ path, target }) {
   return [
     relativeFile({ path, target }),
-    'https://github.com/Homebrew/homebrew-core/blob/master/Formula/' + target,
+    '{BASE_URL}/Homebrew/homebrew-core/blob/master/Formula/' + target,
```

Comment:
I think this one should stay as https://github.com, since that's the canonical host for Homebrew and there's no guarantee that other hosts would have it mirrored.